src/lib/components/inbox/conversations/__tests__/conversations-tests.tsx

```diff
@@ -13,57 +13,57 @@ const meProps = {
-                from_name: "Jean-Luc Collecteur",
-                from_email: "luc+messaging@artsymail.com",
-                to_name: "ACA Galleries",
-                last_message: "Karl and Anna... Fab!",
-                created_at: "2017-06-01T14:14:35.538Z",
-                artworks: [
-                {
-                    id: "bradley-theodore-karl-and-anna-face-off-diptych",
-                    href: "/artwork/bradley-theodore-karl-and-anna-face-off-diptych",
-                    title: "Karl and Anna Face Off (Diptych)",
-                    date: "2016",
-                    artist: {
-                    name: "Bradley Theodore",
-                    },
-                    image: {
-                    url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/normalized.jpg",
-                    image_url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/:version.jpg",
-                    },
+                node: {
+                    id: "582",
+                    inquiry_id: "59302144275b244a81d0f9c6",
+                    from_name: "Jean-Luc Collecteur",
+                    from_email: "luc+messaging@artsymail.com",
+                    to_name: "ACA Galleries",
+                    last_message: "Karl and Anna... Fab!",
+                    created_at: "2017-06-01T14:14:35.538Z",
+                    artworks: [
+                        {
+                            id: "bradley-theodore-karl-and-anna-face-off-diptych",
+                            href: "/artwork/bradley-theodore-karl-and-anna-face-off-diptych",
+                            title: "Karl and Anna Face Off (Diptych)",
+                            date: "2016",
+                            artist: {
+                                name: "Bradley Theodore",
+                            },
+                            image: {
+                                url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/normalized.jpg",
+                                image_url: "https://d32dm0rphc51dk.cloudfront.net/bJ9I_vJX9ksaKFJAkOAIKg/:version.jpg",
+                            },
+                        },
+                    ],
                 },
-                ],
-            },
             },
             {
-            node: {
-                id: "581",
-                inquiry_id: "593020be8b3b814f9f86f2fd",
-                from_name: "Jean-Luc Collecteur",
-                from_email: "luc+messaging@artsymail.com",
-                to_name: "David Krut Projects",
-                last_message: "Hi, I’m interested in purchasing this work. \
+                node: {
+                    id: "581",
+                    inquiry_id: "593020be8b3b814f9f86f2fd",
+                    from_name: "Jean-Luc Collecteur",
+                    from_email: "luc+messaging@artsymail.com",
+                    to_name: "David Krut Projects",
+                    last_message: "Hi, I’m interested in purchasing this work. \
```

Comment:
Nah, this should get fixed whenever we move to prettier - bet theres a tonne of these. TSLint only cares that indentation is consistent, not 100% correct.